src/sql/sqlops.proposed.d.ts

```diff
@@ -336,6 +367,16 @@ declare module 'sqlops' {
 				 * Any additional buttons that should be displayed
 				 */
 				customButtons: Button[];
+
+				/**
+				 * Whether the dialog's content is valid
+				 */
+				readonly valid: boolean;
+
+				/**
+				 * Fired whenever the dialog's valid property changes
+				 */
+				readonly onValidityChanged: vscode.Event<boolean>;
```

Comment:
should we return any error message?

Reply:
I was thinking of maybe adding that in a separate PR if dialog authors end up wanting it, does that sound good or do you think we should include it to start?